### ISSUE TITLE
Auto run at start?

### ISSUE THREAD

**Issue by Aphexus** [sieukrem/jn-npp-plugin#115]
There doesn't seem to be a way to autorun scripts at load time. E.g., to automatically add a menu item and code when starting npp. 

What would be cool, say, if jn could integrate with the favorites plugin so one could keep a list of js scripts and set them to run at load. Of course I think having win api x64 support would be nice.

**Comment by sieukrem** [comment]
@Aphexus I can not follow the request! What I understand is, you want to execute code at the start of npp, to insert new additional menu items. This is the way it works today! There is includes folder. All scripts from it will be executed at start.

**Comment by Aphexus** [comment]
Ok, Thanks, I didn't find this info. ""jN\includes" Folder contains several JavaScript Files. They implement some functionality."

You could add some more info about startup scripts there.

Would it be possible to make it easier to disable scripts without having to comment out everything or move the file out of includes?

E.g., 
    // ##jNDisableScript 
at the top 

The point with the explorer/favorites plugin is that it could be easier to enable and disable scripts by providing a file manager interface to the scripts and making it easy to disable and enable(RMB say).  That is, it already provides an a way to manage files in notepad. Maybe you could talk to the maintainer about it.

https://github.com/funap/npp-explorer-plugin

[old stuff:
https://sourceforge.net/projects/npp-explorer/
https://sourceforge.net/p/npp-plugins/discussion/670933/]

Basically what I'm thinking is that one can use the explorer to "Add script to Startup" "Run Script" when RMB on js files.
